CHANGELOG-core.md

```diff
@@ -16,6 +16,7 @@ This release contains a number of features aimed at improving the node's reliabi
 - New prometheus metrics for the head tracker:
   - `head_tracker_heads_in_queue` - The number of heads currently waiting to be executed. You can think of this as the 'load' on the head tracker. Should rarely or never be more than 0.
   - `head_tracker_callback_execution_duration` - How long it took to execute all callbacks. If the average of this exceeds the time between blocks, your node could lag behind and delay transactions.
+- `ethtx` tasks now support a new parameter, `minRequiredOutgoingConfirmations` which allows you to tune how many confirmations are required before moving on from an `ethtx` task on a per task basis (only works with BulletproofTxManager). If it is not supplied, the default of `MIN_OUTGOING_CONFIRMATIONS` is used (same as the old behaviour).
```

Comment:
I don't understand the difference in behaviour between `minRequiredOutgoingConfirmations` and `MIN_OUTGOING_CONFIRMATIONS`?

From what I can read in this diff https://github.com/smartcontractkit/chainlink/pull/2997/files#diff-b97fb224c64cd286dcf343e0bcae3203R423-R425 they do the same thing, one is global and one is configurable per task. 

Can they have the same name then?! so `minOutgoingConfirmations`?

Reply:
`minRequiredOutgoingConfirmations` is a more descriptive name IMO.

We cannot rename `MIN_OUTGOING_CONFIRMATIONS` because it is a historically used env var. Two options:

1. Rename `minOutgoingConfirmations()` to `minRequiredOutgoingConfirmations()` (but keep the old env var name
2. Rename `minRequiredOutgoingConfirmations` to `minOutgoingConfirmations` everywhere

Which do you prefer?

Reply:
1.